11-multi-track/bb_models_collections/readme.md

```diff
@@ -251,8 +272,19 @@ var Grumble = Backbone.Model.extend({
 })
 ```
 
-## Backbone Collections (30/150)
-What is a backbone collection? A collection acts as an intelligent wrapper for like models. It provides a set of methods for performing the CRUD operations on models of the collection. Create and edit the file `js/collections/grumbles.js`:
+## You-DO Play with BB Models (5 mins)
+With a partner, take 5 minutes to play around with our new Model.
+Reference the [docs](http://backbonejs.org/#Model) to see all available methods for Backbone models.
+
```

Comment:
Any suggestions on how they can go about "playing" with BB models during this exercise? Any properties they should explore in the console?